example/src/app/components/malware-modal/malware-modal.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input, NgZone } from '@angular/core';
+import { SuspiciousAppsService } from '../../services/suspicious-apps.service';
+
+@Component({
+  selector: 'malware-modal',
+  templateUrl: './malware-modal.component.html',
+  styleUrls: ['./malware-modal.component.css'],
+})
+export class MalwareModalComponent {
+  @Input() isDisabled: boolean = false;
```

Comment:
Keď toto ide ako input, tak by nebolo lepši zobraziť malware modal v nejakej parent componente iba keď je enabled? Súvisí to aj s tým čo je v HTML template tento componenty.